features/nanostack/mbed-mesh-api/mbed-mesh-api/NanostackEMACInterface.h

```diff
@@ -1,29 +0,0 @@
-/*
- * Copyright (c) 2016 ARM Limited. All rights reserved.
- * SPDX-License-Identifier: Apache-2.0
- * Licensed under the Apache License, Version 2.0 (the License); you may
- * not use this file except in compliance with the License.
- * You may obtain a copy of the License at
- *
- * http://www.apache.org/licenses/LICENSE-2.0
- *
- * Unless required by applicable law or agreed to in writing, software
- * distributed under the License is distributed on an AS IS BASIS, WITHOUT
- * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
- * See the License for the specific language governing permissions and
- * limitations under the License.
- */
-
-#ifndef NANOSTACKEMACINTERFACE_H
-#define NANOSTACKEMACINTERFACE_H
-
-#include "MeshInterfaceNanostack.h"
-#include "NanostackEthernetPhy.h"
-
-class NanostackEMACInterface : public Nanostack::Interface {
```

Comment:
By this and the other removed class/headers do we make it a breaking change?

Reply:
I don't believe there are any breaking changes for "real code" here.

These two files were a stray remnant of a design that wasn't followed through, and committed by mistake - the corresponding cpp files were missing, so the classes weren't constructible.

It would technically be a breaking change if someone had been including the files for no reason.